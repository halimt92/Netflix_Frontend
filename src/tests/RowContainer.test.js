import React from 'react';
import { shallow } from 'enzyme';
import RowContainer from '../container/row-container/RowContainer';
import Row from '../components/row/Row';

describe('RowContainer', () => {
  it('should render a <Row /> component', () => {
    const wrapper = shallow(<RowContainer />);
    expect(wrapper.find(Row)).toHaveLength(8);
  });

  it('matches RowContainer snapshot', () => {
    const wrapper = shallow(<RowContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
