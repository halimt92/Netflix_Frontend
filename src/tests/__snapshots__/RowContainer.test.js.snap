// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches App snapshot 1`] = `ShallowWrapper {}`;

exports[`RowContainer matches App snapshot 1`] = `ShallowWrapper {}`;

exports[`RowContainer matches RowContainer snapshot 1`] = `ShallowWrapper {}`;

exports[`matches App snapshot 1`] = `ShallowWrapper {}`;
